### DISCUSSION TITLE
How to delete a scheduled event using API?

### DISCUSSION THREAD

**Discussion by sakhmedbayev** [hasura/graphql-engine#7886] [Question]
Disclaimer: I am new to Hasura. I think I am missing some key understanding of how Hasura works. 

Here is the list of steps I did so far:

1. Initiazed a new Hasura project using Heroku Postgresql database
2. using `/v1/query` and the following post body, I managed to create a scheduled event (I see it in the Hasura Web Console):

```
{
   type: "create_scheduled_event",
      args: {
        webhook: "some API endpoint",
        schedule_at: "somedate",
        headers: [
          { name: "method", value: "POST" },
          { name: "Content-Type", value: "application/json" },
        ],
        payload: "somepayload",
        comment: "I SUPPLY A UNIQUI ID TO USE IN THE FOLLOWING DELETE QUERY",
        retry_conf: {
          num_retries: 3,
          timeout_seconds: 120,
          tolerance_seconds: 21675,
          retry_interval_seconds: 12,
        }
      }
}
```

Now, I am trying to delete this event using this post body:

```
{
   type: "delete",
      args: {
        table: {
          schema: "hdb_catalog",
          name: "hdb_scheduled_events",
        },
        where: {
          comment: {
            $eq: `HERE I PROVIDE THE UNIQUE ID I SET ON THE EVENT CREATION ABOVE`,
          }
        }
      }
}
```

and getting back this response:

```
data: {
    path: '$.args',
    error: 'table "hdb_catalog.hdb_scheduled_events" does not exist',
    code: 'not-exists'
  }
```

as I understand `hdb_catalog` is the schema that I should work against but it is not appear anywhere in my Heroku database. I actually managed to create a scheduled event even without any database connected to the project. So, it seems that Hasura uses something else to store my scheduled events, but what??? How to access that database/table? Would you please help me?

**Comment by robhayesmba** [comment]
Couldn't you simply use the `delete_scheduled_event` api described here - https://hasura.io/docs/latest/graphql/core/api-reference/metadata-api/scheduled-triggers.html#delete-scheduled-event?
